tests/sentry/integrations/github/test_integration.py

```diff
@@ -212,27 +225,32 @@ def test_disables_plugin_when_fully_migrated(self):
             url='https://github.com/Test-Organization/foo',
             provider='github',
             external_id=123,
+            config={
+                'name': 'Test-Organization/foo',
+            },
         )
 
         assert 'github' in [p.slug for p in plugins.for_project(project)]
 
-        self.assert_setup_flow()
+        with self.tasks():
+            self.assert_setup_flow()
 
         assert 'github' not in [p.slug for p in plugins.for_project(project)]
 
     @responses.activate
     def test_basic_flow(self):
-        self.assert_setup_flow()
+        with self.tasks():
+            self.assert_setup_flow()
 
         integration = Integration.objects.get(provider=self.provider.key)
 
         assert integration.external_id == self.installation_id
         assert integration.name == 'Test Organization'
         assert integration.metadata == {
-            'access_token': self.access_token,
+            'access_token': None,
             # The metadata doesn't get saved with the timezone "Z" character
             # for some reason, so just compare everything but that.
-            'expires_at': self.expires_at[:-1],
+            'expires_at': None,
```

Comment:
for this and the `access_token` change: we update this in the gh client's `get_token` method, which is no longer happening after the removal of the `get_repositories` call